src/main/java/com/ocelotslovebirds/birdhaus/datagen/BlockTagsGen.java

```diff
@@ -24,20 +22,13 @@ public BlockTagsGen(DataGenerator generator, ExistingFileHelper helper) {
      * Possibility to add a tag to a block. Unused. BlockEntity does a better job of implementing complex behaviors.
      */
     @Override
-    protected void addTags() {
-        tag(BlockTags.PLANKS)
-                .add(Blocks.OAK_PLANKS, Blocks.SPRUCE_PLANKS, Blocks.BIRCH_PLANKS, Blocks.JUNGLE_PLANKS,
-                        Blocks.ACACIA_PLANKS, Blocks.DARK_OAK_PLANKS, Blocks.CRIMSON_PLANKS, Blocks.WARPED_PLANKS);
-        tag(BlockTags.LOGS)
-                .add(Blocks.OAK_LOG, Blocks.SPRUCE_LOG, Blocks.BIRCH_LOG, Blocks.JUNGLE_LOG,
-                        Blocks.ACACIA_LOG, Blocks.DARK_OAK_LOG, Blocks.CRIMSON_STEM, Blocks.WARPED_STEM);
-    }
+    protected void addTags() {}
 
```

Comment:
Tagging the blocks individually proved to be unnecessary since BlockTags.PLANKS and BlockTags.LOGS already included all types of planks and logs in the game. @shafiqimtiaz

Reply:
Ok. That simplifies a lot. Thanks 👍🏽